examples/3d_examples/MagneticCiliaCarpetCase/immersed_magnetic_cilia_carpet.py

```diff
@@ -219,18 +271,40 @@ def update_carpet_lag_grid_fields():
     carpet_base_centroid = np.array(
         [0.5 * domain_x_range, 0.5 * domain_y_range, 0.1 * domain_z_range]
     )
+    n_elem_per_rod = int(grid_size_x * rod_elem_prefactor / num_rods_along_x)
     cilia_carpet_simulator = MagneticCiliaCarpetSimulator(
+        magnetic_elastic_ratio=magnetic_elastic_ratio,
```

Comment:
lets call this magnetic Bond number?

Reply:
[ce91966](https://github.com/SophT-Team/SophT/pull/120/commits/ce919664854a349d4b924b8ee527419355988c09)